bin/gwd/request.ml

```diff
@@ -775,7 +765,8 @@ let treat_request =
           w_base @@ WiznotesDisplay.print_search
         | _ ->
             w_base @@ fun conf base ->
-            incorrect_request conf base ~comment:"error #10"
+            let _str = Format.sprintf "m=%s is not available here" m in
+            incorrect_request conf base ~comment:"m=FORUM is not available here"
```

Comment:
There is no reason to use hardcoded comment with FORUM, while _str string built just above, to cover all supported incorrect request.